examples/dyn_proxy.rs

```diff
@@ -0,0 +1,59 @@
+#![deny(warnings)]
+use warp::hyper::body::{Body, Bytes};
+use warp::hyper::{Client, Request};
+use warp::{
+    http::{method::Method, HeaderMap, Response},
+    path::FullPath,
+    Filter, Rejection,
+};
+
+static PROXY_TARGET: &'static str = "http://httpbin.org";
+
+async fn proxy_request(
+    method: Method,
+    path: FullPath,
+    headers: HeaderMap,
+    body: Bytes,
+) -> Result<Response<Body>, Rejection> {
+    let request = build_request(&method, &path, &headers, body);
+    let client = Client::new();
+    let response = client.request(request).await.unwrap();
```

Comment:
It's likely eventually there will be some network error sending the request, it'd probably be better to return a 503 response in that case. Otherwise, the unwrap will crash the proxy server.

Reply:
Good catch, fixed that.